miden-crypto/src/lib.rs

```diff
@@ -49,6 +48,20 @@ pub type Map<K, V> = alloc::collections::BTreeMap<K, V>;
 #[cfg(not(feature = "hashmaps"))]
 pub use alloc::collections::btree_map::Entry as MapEntry;
 
+/// An alias for a simple set.
+///
+/// By default, this is an alias for the [`alloc::collections::BTreeSet`]. However, when the
+/// `hashmaps` feature is enabled, this becomes an alias for hashbrown's HashSet.
+#[cfg(feature = "hashmaps")]
+pub type Set<V> = hashbrown::HashSet<V>;
```

Comment:
note/nit: `hashbrown` is the `stdlib`'s actual implementation, we will move away from it soon~ish